bin/calibrate.py

```diff
@@ -60,7 +60,8 @@ def main(argv=None):
         while good < 3:
             p = subprocess.Popen(command,
                                  stdout=subprocess.PIPE,
-                                 stderr=subprocess.STDOUT)
+                                 stderr=subprocess.STDOUT,
+                                 universal_newlines=True)
```

Comment:
why universal newlines?

Reply:
We're talking to a command that is going to give us back text and is going to be handled as such, but in Py3 `subprocess` stdout/stderr comes back as bytes unless `universal_newlines` is set. The other alternative is to `decode()` it, but this seems to state the intent a little more cleanly (even better if we could say `text=True` but that doesn't work as long as we need 2.7 compat since it's a new name)